packages/frontend/component/src/ui/slider/slider.tsx

```diff
@@ -0,0 +1,101 @@
+import type { KeyboardEvent, MouseEvent } from 'react';
+import { useCallback, useEffect, useMemo, useRef, useState } from 'react';
+
+import * as styles from './index.css';
+
+export interface SliderProps {
+  value?: number;
+  onChange?: (value: number) => void;
+  min?: number;
+  max?: number;
+  step?: number;
+  nodes?: number[]; // The values where the nodes should be placed
+}
+
+export const Slider = ({
```

Comment:
Impl with `@radix-ui/slider` for better accessibility like mobile touch and keyboard support.

Reply:
I have tried using Radix-UI, but the thumb does not move proportionally, causing the stepNode and thumb to be misaligned. This issue is currently blocking me from using Radix-UI. We can discuss some potential solutions.

Reply:
The specific situation is that the stepNode is at [0%, 25%, 50%, 100%], whereas the thumb's movement is at [0%+10px, 25%+5px, 50%, 100%-5px], with this additional 5px increment being calculated through a certain logic based on the thumb's width. This is quite perplexing to me.

Reply:
Not sure about this but I think styling should not be that tricky.
I tried your storybook locally and tbh it works very badly ...